src/amo/components/AddonSummaryCard/index.js

```diff
@@ -58,7 +58,7 @@ export const AddonSummaryCardBase = ({
     addonAverage = i18n.sprintf(
       // eslint-disable-next-line max-len
       // translators: averageRating is a localized number, such as 4.5 in English or ٤٫٧ in Arabic.
-      i18n.gettext('%(averageRating)s star average'),
+      i18n.gettext('%(averageRating)s Stars out of 5'),
```

Comment:
Looks Good to me. :100:

Reply:
If there is only 1 star, it should read: "1 Star out of 5". 

You can look at how other files use `ngettext` to show singular and plural forms of the text (or reach out if you need help :)

Reply:
@rebmullin  I have updated the changes with `ngettext` and added a test case for it.  Since  `ngettext` accepts number type as the third parameter, I've modified the `averageRating` and `roundedAverage`(Before `averageRating` was in string type). Let me know how it looks now.